3_navigation.ipynb

```diff
@@ -109,26 +226,73 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Return to the RViz screen and give \"2D Nav Goal\" to move the HSR autonomously.\n",
```

Comment:
"Return to the RViz screen and use \"2D Nav Goal\" to move the HSR autonomously.\n",